src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -446,12 +497,31 @@ public void addTab(@NonNull BibDatabaseContext databaseContext, boolean raisePan
 
     public void addTab(@NonNull LibraryTab libraryTab, boolean raisePanel) {
         tabbedPane.getTabs().add(libraryTab);
+
         if (raisePanel) {
             tabbedPane.getSelectionModel().select(libraryTab);
             tabbedPane.requestFocus();
         }
 
         libraryTab.setContextMenu(createTabContextMenuFor(libraryTab));
+
+        // Ensure layout is updated
+        updateContent();
+        updateSidePane();
+
+        // Set the preferred width and divider position explicitly
+        sidePane.setPrefWidth(260);
```

Comment:
`260` is a new value. Should be stored as constant in this class. - Why is `sidePane` used and not `SplitPane`?